engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowLifecycleActor.scala

```diff
@@ -65,6 +66,12 @@ trait WorkflowLifecycleActor[S <: WorkflowLifecycleActorState] extends LoggingFS
   def successResponse: WorkflowLifecycleSuccessResponse
   def failureResponse(reasons: Seq[Throwable]): WorkflowLifecycleFailureResponse
 
+  override def supervisorStrategy = OneForOneStrategy() {
+    case ex: ActorInitializationException =>
+      context.parent ! failureResponse(Seq(ex))
+      Stop
```

Comment:
This is a little confusing: 

I think we also want to stop this actor, and any other supervisees other than the one which failed?

Or will that happen naturally when the parent is stopped?